src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACPackagedTerminalAirConditioner.cpp

```diff
@@ -350,6 +351,9 @@ namespace energyplus {
             _heatExchanger->setString(HeatExchanger_AirToAir_SensibleAndLatentFields::ExhaustAirOutletNodeName, coolingCoilOutletNodeName);
           }
         }
+      } else if (_coolingCoil->iddObject().type() == IddObjectType::Coil_Cooling_DX) {
+        _coolingCoil->setString(Coil_Cooling_DXFields::EvaporatorInletNodeName, coolingCoilInletNodeName);
+        _coolingCoil->setString(Coil_Cooling_DXFields::EvaporatorOutletNodeName, coolingCoilOutletNodeName);
```

Comment:
Need to add FT to support Coil:Cooling:DX on ZoneHVAC:PackagedTerminalAirConditioner.